Sources/ArgumentParser/Parsable Properties/ArgumentHelp.swift

```diff
@@ -11,6 +11,17 @@
 
 /// Help information for a command-line argument.
 public struct ArgumentHelp {
+  public enum Visibility {
```

Comment:
Could you add an abstract for this type, as well?

Reply:
Sure !